README.md

```diff
@@ -16,9 +16,9 @@ $ sbt server/run
 ```
 
 After downloading a bunch of packages (these are cached locally, so this will only be slow the first time),
-you'll have a server running on `localhost:9000`. It will try to connect to a labrad manager running on
+you'll have a server running on `localhost:7667`. It will try to connect to a labrad manager running on
 the same machine, so you should start up a labrad manager if you want to see anything interesting.
-Note that the server uses the [play framework](https://www.playframework.com/) which
+Note that the server uses a json library from the [play framework](https://www.playframework.com/) which
```

Comment:
Is it still true that you need java 8 even though we're just importing one library?